docs/src/pages/components/visuallyhidden/react.mdx

```diff
@@ -1,25 +1,29 @@
-import { Button, IconDone, VisuallyHidden } from '@aws-amplify/ui-react';
-import { Example } from '@/components/Example';
+import { Button, IconDone, VisuallyHidden, View } from '@aws-amplify/ui-react';
```

Comment:
```suggestion
import { Button, IconDone, VisuallyHidden } from '@aws-amplify/ui-react';
```

looks like `View` is not in use anywhere and could add a line break below this import